apps/mobile/src/components/ProcessedText/ProcessedTextAsPlaintext.tsx

```diff
@@ -0,0 +1,35 @@
+import { Text, TextProps } from 'react-native';
+
+import { Typography } from '~/components/Typography';
+import GalleryProcessedText from '~/shared/components/GalleryProccessedText/GalleryProcessedText';
+
+import { MentionComponent } from './elements/MentionComponent';
+import { TextComponent } from './elements/TextComponent';
+
+type ProcessedTextProps = {
+  text: string;
+} & TextProps;
+
+export default function ProcessedTextAsPlaintext({ text, ...props }: ProcessedTextProps) {
```

Comment:
looks like this is mostly the same as the existing `ProcessedText` component.

could we instead use that component and add an existing prop, like `plaintextOnly`?